src/metabase/models/table.clj

```diff
@@ -151,48 +151,3 @@
   [table-id]
   {:pre [(integer? table-id)]}
   (db/select-one-field :db_id Table, :id table-id))
-
-
-;;; ------------------------------------------------------------ Persistence Functions ------------------------------------------------------------
-
-(defn retire-tables!
```

Comment:
why are we removing this? Is it because this was only called from the Raw/introspect whatnotery?